docs/source/core_concepts/index.rst

```diff
@@ -193,13 +193,13 @@ In k2, you would use the following code to compute it:
     fsa = k2.Fsa.from_str(s)
     fsa.draw('fsa2.svg')
     fsa = k2.create_fsa_vec([fsa])
-    total_scores = k2.get_tot_scores(fsa, log_semiring=False, use_double_scores=False)
+    total_scores = k2.Fsa.get_tot_scores(fsa, log_semiring=False, use_double_scores=False)
```

Comment:
I think it should be `fsa.get_tot_scores(log_semiring=False, use_double_scores=False)`